### DISCUSSION TITLE
How to enable / disable options based on another select option in livewire

### DISCUSSION THREAD

**Discussion by MAdnanNaeem** [livewire/livewire#5113] [1. Help]
Wire:model removes the  'disabled' attribute from the select field. Can anyone suggest how i can " enable / disable options based on another select option in livewire" ?

Thanks in advance.

**Comment by joshhanley** [comment]
@MAdnanNaeem can you share your code as Livewire should not be removing the `disabled` attribute?

Livewire does control and remove the `selected` attribute, but `disabled` shouldn't be touched. I've done this in some of my apps in the past and it seemed to work for me.

Thanks!
